static/css/style.css

```diff
@@ -45,16 +45,17 @@ body {
 .container .credit {
 	margin: 20px 0;
 }
-
```

Comment:
Well long story. At one point I did some changes in my style.css and someone else did on the main repository changes to the same line. Then I had to resolve the conflict in my local git repository and ended up removing with the -f option the style.css file in my local git. After that I cut and pasted the style css from the main repository and good old VIM did some rock and roll on my, yes you guessed indentation :smile: